src/ontology/edits/oeo-model.omn

```diff
@@ -1319,9 +1301,36 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1391",
         <http://purl.obolibrary.org/obo/RO_0002234> some OEO_00020013
     
     
+Class: OEO_00010296
+
+    
 Class: OEO_00010362
 
     
+Class: OEO_00010404
+
+    
+Class: OEO_00010405
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An energy balance collection is a dataset of energy balances for various time steps and/or temporal region.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "An energy balance collection is a dataset of energy balances for various time steps and/or temporal regions.",
```

Reply:
decide for one spelling of dataset vs data set
dataset -> data set ## if accepted, also applies to the `synthetic dataset` definition
if not, then edit definition of `energy balance`

Reply:
Yes, this should be harmonised. I accept this proposal and change the other classes.